core/src/main/java/org/pentaho/agilebi/modeler/nodes/AbstractMetaDataModelNode.java

```diff
@@ -177,6 +186,49 @@ public void setClassname( String classname ) {
     }
   }
 
+  @Bindable
```

Comment:
getter/setter for altText same as other functions see #getClassname() and #setClassname( String classname )